style.css

```diff
@@ -398,3 +398,110 @@ span {
   background: #fff;
   margin-bottom: 30px;
 }
+
+.desktop-tools {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-link {
+    display: none;
+  }
+
+  .nav-container {
+    padding: 10px 40px;
+  }
+
+  .desktop-tools {
+    display: inline;
+  }
+
+  .display-tools,
+  li,
+  a {
+    padding-right: 10px;
+    height: 0%;
+  }
+
+  .major-container {
+    padding: 40px 50px;
+  }
+
+  .project-card {
+    display: grid;
+    padding: 10px;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .project-card img {
+    height: 100%;
+    width: 100%;
+  }
+
+  .prject-card-info {
+    padding: 20px;
+  }
+
+  .order-2 {
+    order: 2;
+  }
+
+  .about-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
```

Comment:
- Hey, let's also make some adjustment to our styling to improve the spacing of the sides of the about section to make our design close to the Figma design as much as we can below:


*your design*
![Screenshot from 2022-12-16 17-15-56](https://user-images.githubusercontent.com/59397159/208142143-151befbe-1364-4853-94d2-271f0632c0b6.png)

*Figma design*
![Screenshot from 2022-12-16 17-14-48](https://user-images.githubusercontent.com/59397159/208142154-478dc801-a26e-41e0-8956-9459a77299e3.png)

Reply:
- Hey, let's make some improvement on the about myself text so the text `myself` goes to the next line. You can try reducing the width or using the `<br>` to achieve that. see below:


*Figma design*
![Screenshot from 2022-12-16 17-29-07](https://user-images.githubusercontent.com/59397159/208145034-9ce34926-1933-4e2e-b2e6-397ffcbfd140.png)

*your design*
![Screenshot from 2022-12-16 17-28-42](https://user-images.githubusercontent.com/59397159/208145043-411d6437-7f98-4a28-a14b-f0babb4bb948.png)